Rakefile

```diff
@@ -2,6 +2,16 @@ require 'rspec/core/rake_task'
 require 'bundler/gem_tasks'
 
 lib_folder = File.expand_path("../lib", __FILE__)
+
+RUBIES = ['ruby-2.0', 'ruby-2.1', 'ruby-2.2', 'ruby-2.3.0']
+
+task :all do |task|
+  RUBIES.each do |ruby_v|
+    puts "\n\n\nUsing #{ruby_v}\n\n\n"
+    puts `bash -lc "rvm use #{ruby_v}; rspec spec"`
```

Comment:
rspec provides a class for defining rake tasks. I think RVM has it's own Ruby API too. Use that. What if the user is not using bash? Or if he does not have bash?